docs/arch/adr-003-versioning-branching-strategy.md

```diff
@@ -0,0 +1,66 @@
+# Release Versioning / Branching Strategy
+
+## Context
+With `dbt` ever evolving and progressing, we continue to ship and release versions on a regular basis. As we release `dbt`, versioning and branching become key to identifying what changes for users and conveying the expectations of those changes. We currently push all our changes to `main` which is the latest version of our code. We don't always want to release the latest version of our code though. We want to be able to isolate changes to go into particular releases based on urgency (e.g. bug fixes) and stability (e.g. new features). 
```

Comment:
> As we release `dbt`, versioning and branching become key to identifying what changes for users and conveying the expectations of those changes.

This sentence uses "changes" as both a noun and a verb in the same sentence and confused me. Maybe use "is different" for the verb?